.github/workflows/build-and-test.yaml

```diff
@@ -0,0 +1,80 @@
+# This job runs the project tests
+
+name: Build and test
+
+# Trigger the workflow on push or pull request to master or develop
+
+on:
+  push:
+    branches: [ develop ]
```

Comment:
i don't think we need the `develop` branch

Reply:
Agreed. Updated.